test_istgt.sh

```diff
@@ -527,6 +528,38 @@ run_replication_factor_test()
 	local replica1_vdev="/tmp/test_vol1"
 	local ret=0
 
+	REPLICATION_FACTOR=1
+	CONSISTENCY_FACTOR=1
+	setup_test_env
+	start_replica -i "$CONTROLLER_IP" -p "$CONTROLLER_PORT" -I "$replica1_ip" -P "$replica1_port" -V $replica1_vdev &
+	replica1_pid=$!
+	sleep 2
+
+	while [ 1 ]; do
+		# We have started istgt with 20 second replica timeout
```

Comment:
actually, we should start istgt with normal timeout so that the condition we added will come into effect.

Reply:
ok.
This comment is confusing since `replica_timeout` doesn't have any relation with `replica status query` interval.
 In testing code, `replication_test` reports healthy status after receiving `replica status query` three times once rebuild starts. Due to this, the replica will take around 120 seconds to become healthy.